mapping/sustainability.yaml

```
@@ -1054,17 +1082,17 @@
           ]
         }
 -   id: '3.19'
-    title: Public consultation meetings
+    title: 'Public consultation meetings '
     module: Climate finance
     indicator: Public participation
-    disclosure format: Disclose the occurrence of public meetings with communities and impacted groups including the minutes, number of the participants, dates and location of these meetings (E.g. Meeting 1 \[date\] \[location\] \[number of participants\] \[Document\], Meeting 2 \[date\] \[location\] \[number of participants\], \[Document\]).
+    disclosure format: Disclose the occurrence of public meetings with communities and impacted groups including the meeting invite, the number of participants, dates and location of these meetings.
```

```suggestion
    disclosure format: Disclose the occurrence of public meetings with communities and impacted groups including the meeting invite, the number of participants, dates and location of these meetings.
```
@mgraca-prado The wording here is slightly different from your suggested text (placement of the word 'the'), but I'd suggest updating the IDS element to the text above, for grammatical reasons.  

@odscjen Shall we hold off on this change till we hear back from Maria?

---

Given the change was just to improve the grammar I think we can go ahead with the change.